apisix/http/router/radixtree_host_uri.lua

```diff
@@ -95,7 +95,8 @@ local function create_radixtree_router(routes)
 
     for _, route in ipairs(routes or {}) do
         local status = core.table.try_read_attr(route, "value", "status")
-        if status and status ~= 1 then
+        -- check the status
+        if status and status == 0 then
```

Comment:
only `if status == 0 then`, this is enough

Reply:
will there be incompatibilities?
such as an existing route in etcd does not have the `status` attribute?

Reply:
You should add test cases for it:)

Reply:
yes, i will